src/main/java/org/opentripplanner/graph_builder/module/PruneFloatingIslands.java

```diff
@@ -38,13 +38,13 @@
      * this field indicate the maximum size for island without stops
```

Comment:
This comment is confusing.  It's a max for pruning; a min for survival.  I think the variable should be renamed.